### ISSUE TITLE
Suggested Posts not blocked 

### ISSUE THREAD

**Issue by edent** [indywidualny/FaceSlim#241]
Not sure if this is a change to Facebook or a bug in 3.0.2

I'm getting several Sponsored Posts in my news feed. I've checked my FaceSlim settings, and they are marked as blocked.

**Comment by indywidualny** [comment]
Yep. Facebook team changes sometimes their sponsored div classes. Something may have been changed.

**Comment by Bitim** [comment]
Maybe you should add this filter:
https://github.com/uBlockOrigin/uAssets/issues/233

**Comment by indywidualny** [comment]
@Bitim It's for full Facebook site. Mobile is a bit different. I will locate them somehow but I'd really like someone to do it and pull request the change. I'm not using Facebook that much to see any ads. Well. I'm not using it at all lately.

This line is responsible for hiding sponsored stuff (plain CSS rule): https://github.com/indywidualny/FaceSlim/blob/d13227d241333ce4980b63eb181d06ab1db43d16/app/src/main/java/org/indywidualni/fblite/webview/MyWebViewClient.java#L199-L202

**Comment by Splike** [comment]
Any update to this? I can't hide sponsored posts with the lines found [here](https://github.com/uBlockOrigin/uAssets/issues/233).

**Comment by jameshilliard** [comment]
The only way to reliably filter facebook ads is using a recursive filtering algorithm such as [this](https://github.com/tiratatp/facebook_adblock/blob/master/src/main.js).
